xlml/utils/gke.py

```diff
@@ -55,10 +56,14 @@ def run_job(
   """
 
   @task
-  def deploy_job():
+  def deploy_job(gcs_location):
+    body['spec']['template']['spec']['containers'][0]['env'].append(
```

Comment:
I don't like modifying `body` like this after the fact. `gcs_location` will have to be plumbed in somewhere else since it's an xcom (or depends on one). 

I'll take another look at the code and get back to you. The change makes sense in principle, and there should be a way to plumb dynamic variables to the environment at test time.

Reply:
Discussed offline: please leave a TODO. I'm doing some refactoring and can add the plumbing we need.